db/migrate/20161130153206_add_last_updated_at_to_content_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastUpdatedAtToContentItems < ActiveRecord::Migration[5.0]
+  def change
+    add_column :content_items, :last_updated_at, :datetime
```

Comment:
I am not sure `public_updated_at` fits our current context.. I am happy that if this turns out wrong rails make is easy to update this..